CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
-- Update chart to [kube-prometheus-stack-44.2.0](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-44.2.0). See [breaking changes](./changelog/32.x_44.x.md)
+- Update chart to [kube-prometheus-stack-44.2.0](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-44.2.0). See [breaking changes](https://github.com/giantswarm/prometheus-operator-app/blob/master/README.md#from-2.x-to-3.x)
```

Comment:
Correct link: https://github.com/giantswarm/prometheus-operator-app/blob/main/README.md#from-2x-to-3x